### PR TITLE
Add script to generate config section for readme

### PR DESCRIPTION
**Pull request by johnjacobkenny**

### CODE REVIEW

**Comment by edgardmessias** [comment]
Add `out/tools/**` in `.vscodeignore`

File: src/tools/generateConfigSectionForReadme.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const packageJsonPath: string = join(cwd(), "package.json");
+const packageJson = JSON.parse(readFileSync(packageJsonPath));
+const properties = packageJson.contributes.configuration.properties;
+
+console.log("|Config|Description|Default|");
+console.log("|-|-|-|");
+
+Object.keys(properties).forEach( val => {
+    let prop = properties[val];
+    console.log(`|\`${val}\`|${prop.description}|\`${prop.default}\`|);
```
Comment (edgardmessias) [inline src/tools/generateConfigSectionForReadme.ts]: @johnjacobkenny, missing "`" before ")"

Comment (johnjacobkenny) [reply src/tools/generateConfigSectionForReadme.ts]: Fixed

File: src/tools/generateConfigSectionForReadme.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { readFileSync } from "fs";
+import { join } from "path";
+import { cwd } from "process";
+
+const packageJsonPath: string = join(cwd(), "package.json");
+const packageJson = JSON.parse(readFileSync(packageJsonPath));
```
Comment (edgardmessias) [inline src/tools/generateConfigSectionForReadme.ts]: Change `readFileSync(packageJsonPath)` to `readFileSync(packageJsonPath, {encoding: 'utf8'})` to fix travis

**Comment by edgardmessias** [comment]
Thanks @johnjacobkenny

**Pull request merged**